_posts/elements/2015-05-12-forms.md

```diff
@@ -38,8 +38,8 @@ info: Forms allow users to interact with the interface, set preferences, and ent
             </div>
             <div class="field">
                 <label class="field-label field-required" for="email_address_default_helper">Email address (default, required text field with helper text)</label>
-                <input class="field-input input-text" type="email" id="email_address_default_helper" name="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper" required>
-                <div class="field-hint" id="email_address_default_helper">
+                <input class="field-input input-text" type="email" id="email_address_default_helper" name="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper_text" required>
```

Comment:
Also, please sync up the names your using for helper/hint text. I'd prefer if everything was using one and I'd recommend ``[id]-hint``.